android/src/main/java/com/magicleap/magicscript/scene/nodes/layouts/params/LinearLayoutParams.kt

```diff
@@ -1,16 +1,27 @@
 package com.magicleap.magicscript.scene.nodes.layouts.params
 
 import com.magicleap.magicscript.scene.nodes.props.Alignment
+import com.magicleap.magicscript.scene.nodes.props.ItemAlignmentMap
+import com.magicleap.magicscript.scene.nodes.props.ItemPaddingMap
 import com.magicleap.magicscript.scene.nodes.props.Padding
 import com.magicleap.magicscript.utils.Vector2
 
 class LinearLayoutParams(
     val orientation: String,
     size: Vector2,
-    itemPadding: Padding,
+    defaultItemPadding: Padding,
     itemHorizontalAlignment: Alignment.HorizontalAlignment,
-    itemVerticalAlignment: Alignment.VerticalAlignment
-) : LayoutParams(size, itemPadding, itemHorizontalAlignment, itemVerticalAlignment)
+    itemVerticalAlignment: Alignment.VerticalAlignment,
+    itemsPadding: ItemPaddingMap?,
+    itemsAlignment: ItemAlignmentMap?
+) : LayoutParams(
+    size = size,
+    defaultItemPadding = defaultItemPadding,
+    itemHorizontalAlignment = itemHorizontalAlignment,
```

Comment:
Now, as we have additional `itemsAlignment` property, I think we should use `defaultItemAlignment` (of type `Alignment`) instead `itemVerticalAlignment` and `itemHorizontalAlignment` separetly.